comps/comps-foreman-rhel7.xml

```diff
@@ -298,8 +298,9 @@
       <packagereq type="default">foreman-installer-katello</packagereq>
       <packagereq type="default">katello-certs-tools</packagereq>
       <packagereq type="default">katello-selinux</packagereq>
-      <packagereq type="default">keycloak-httpd-client-install</packagereq>	      
+      <packagereq type="default">keycloak-httpd-client-install</packagereq>
       <packagereq type="default">mod_passenger</packagereq>
+      <packagereq type="default">nodejs-theforeman-env</packagereq>
```

Comment:
This is unexpected. It looks like this added it in the non-scl style while nowadays it needs to be in SCL. Perhaps you based it off an old branch? It is the reason for the failure though.

Reply:
@ekohl The first time I attempted to add `@theforeman/env`, it errored near the end because it couldn't find a Python module called `click`.  I installed `pip3` and the `click` module, did a `git stash` to return to the previous state before adding `env`, and then tried again.  Perhaps it was something to do with that flow..

Reply:
built again, now failing differently